pages/getting-started/sending-your-first-email.mdx

```diff
@@ -74,31 +205,43 @@ Hello friend!
 {% endif %}
 ```
 
+In this example, the first two lines state that if a subscriber has the `first_name` object, the opening line of the email should greet the subscriber using that name. If the subscriber's `first_name` is Alice, the opening line will read "Hello Alice!" The third and fourth lines specify that if there is not a `first_name` object defined, the opening line of the email should read "Hello friend!" 
+
 ## Sending emails based on an RSS feed
 
-You'll want to get set up with our [RSS automation](/advanced-features/rss-to-email).
+To learn about using an RSS feed, read [RSS automation](/advanced-features/rss-to-email).
+
+## Things to do after you send your first email
+
+So you've sent your newsletter's first email. That's great! What happens next?
+
+### Update your email's slug
 
-## After sending
+The "slug" is the end of a URL. Each email has a unique slug. You can change the slug of an email if you need to. 
 
-### Updating your email's slug
+If you update the slug of an email that you've already sent, don't worry! The old link that references an old slug still works. Buttondown keeps track of historical slugs for each email and redirects readers who go to the old link accordingly.
 
-If you update the slug of an email that's already been sent out, worry not! The old link that references an old slug will still work; Buttondown keeps track of historical slugs for each email and redirects readers who go to the old link accordingly.
+<!-- How do you do this? I couldn't find it in the UI at all. If it's an API-only feature, we should probably remove this from the docs, or link to the API docs. -->
 
-### Updating your email's issue number
+### Update your email's issue number
 
-Buttondown tries to be clever with determining what issue number your newsletter is on. Sometimes it is either _too_ clever or not nearly clever enough, though, and you end up in a situation where your seventh issue is showing up as your eighth or vice-versa. This is particularly pernicious as a problem if you're bringing in archives from another system.
+Buttondown tries determine what issue number your newsletter is on. For example, your first email will be labeled number 1. But it doesn't always get this right. This is particularly a problem for people who have imported the archives of an existing newsletter from another system. If you need to, you can manually edit your issue numbers in your Archive page. 
 
-You can manually edit your issue numbers by going to the **Edit Email** screen in your Archive page. The **ID** field corresponds with the issue number:
+<!-- TKTK This is another thing that wants a procedure, but I can't find the UI. -->
```

Comment:
@jmduke I couldn't get this modal to appear anywhere in the emails page, or by searching for it. How do I find it?

Reply:
That doc is straight up wrong and has been for what is, in retrospect, a disturbing amount of time. You change it now by going to "edit" (which brings you back to the main screen where you write the email) and then clicking on "Web archive".